src/ProcessManager/ChromeManager.php

```diff
@@ -37,6 +42,21 @@ public function __construct(?string $chromeDriverBinary = null, ?array $argument
         $this->arguments = $arguments ?? $this->getDefaultArguments();
     }
 
+    public function setExtensions(array $extensions): void
```

Comment:
Can you add new constructor arguments instead of these new methods? We try to keep our objects as "immutable" as possible.